### DISCUSSION TITLE
add option: border-x- and border-y- to border-width

### DISCUSSION THREAD

**Discussion by KareemDa** [tailwindlabs/tailwindcss#5935] [Ideas]
**What version of Tailwind CSS are you using?**
v2.0.4

**What build tool (or framework if it abstracts the build tool) are you using?**
@nuxtjs/tailwindcss

**What version of Node.js are you using?**
v14.16.0

**Describe your issue**
add option new options: 
- `border-x-1`
- `border-x-2`
- `border-x-4`
- `border-x-8`
- `border-y-1`
- `border-y-2`
- `border-y-4`
- `border-y-8`

**Comment by KareemDa** [comment]
I can make a PR to add this feature

**Comment by adamwathan** [comment]
Hey! This already exists in the next version of Tailwind: https://github.com/tailwindlabs/tailwindcss/pull/5639

_(Also, please use discussions for feature requests, not issues, thanks 🙏 )_

**Reply by KareemDa** [reply]
thanks. I'm really interested in contributing with tailwindcss. 
can I help in and get tasks in v.3 alpha?
